frontend/test/metabase/scenarios/dashboard/subscriptions.cy.spec.js

```diff
@@ -115,6 +124,79 @@ describe("scenarios > dashboard > subscriptions", () => {
       // Implicit assertion (word mustn't contain string "null")
       cy.findByText(/^Emailed monthly on the first (?!null)/);
     });
+
+    it.skip("should work when using dashboard default filter value on native query with required parameter (metabase#15705)", () => {
+      // In order to reproduce this test, we need to use the old syntac for dashboard filters
+      mockSessionProperty("field-filter-operators-enabled?", false);
+
+      cy.createNativeQuestion({
+        name: "15705",
+        native: {
+          query: "SELECT COUNT(*) FROM ORDERS WHERE QUANTITY={{qty}}",
+          "template-tags": {
+            qty: {
+              id: "3cfb3686-0d13-48db-ab5b-100481a3a830",
+              name: "qty",
+              "display-name": "Qty",
+              type: "number",
+              required: true,
+            },
+          },
+        },
+      }).then(({ body: { id: QUESTION_ID } }) => {
+        cy.createDashboard("15705D").then(({ body: { id: DASHBOARD_ID } }) => {
+          // Add filter to the dashboard
+          cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+            // Using the old dashboard filter syntax
+            parameters: [
+              {
+                name: "Quantity",
+                slug: "quantity",
+                id: "930e4001",
+                type: "category",
+                default: "20",
+              },
+            ],
+          });
+
+          // Add question to the dashboard
+          cy.request("POST", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+            cardId: QUESTION_ID,
+          }).then(({ body: { id: DASH_CARD_ID } }) => {
+            // Connect filter to that question
+            cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+              cards: [
+                {
+                  id: DASH_CARD_ID,
+                  card_id: QUESTION_ID,
+                  row: 0,
+                  col: 0,
+                  sizeX: 12,
+                  sizeY: 10,
+                  parameter_mappings: [
+                    {
+                      parameter_id: "930e4001",
+                      card_id: QUESTION_ID,
+                      target: ["variable", ["template-tag", "qty"]],
+                    },
+                  ],
+                },
+              ],
+            });
+          });
+          assignRecipient({ dashboard_id: DASHBOARD_ID });
+        });
+      });
+      // Click anywhere outside to close the popover
+      cy.findByText("15705D").click();
+      cy.findByText("Send email now").click();
+      cy.findByText("Email sent");
+      cy.request("GET", "http://localhost:80/email").then(({ body }) => {
+        expect(body[0].html).not.to.include(
+          "An error occurred while displaying this card.",
```

Comment:
At some point, you probably want to make this into a command and always include the check that it shouldn't include error, and a positive assertion when known/possible.

Reply:
Absolutely!
I'm thinking about it every time I use this, but also trying to see which patterns will emerge during these early stages of email testing so I can make better abstractions later.